src/main/java/uk/gov/hmcts/reform/blobrouter/data/EnvelopeSummaryRepository.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.blobrouter.data;
+
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Repository;
+import uk.gov.hmcts.reform.blobrouter.data.model.EnvelopeSummary;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.List;
+
+@Repository
+public class EnvelopeSummaryRepository {
+
+    private final NamedParameterJdbcTemplate jdbcTemplate;
+    private final EnvelopeSummaryMapper mapper;
+
+    public EnvelopeSummaryRepository(
+        NamedParameterJdbcTemplate jdbcTemplate,
+        EnvelopeSummaryMapper mapper
+    ) {
+        this.jdbcTemplate = jdbcTemplate;
+        this.mapper = mapper;
+    }
+
+    public List<EnvelopeSummary> find(Instant from, Instant to) {
+        return jdbcTemplate.query(
+            "SELECT e.*, ev2.* "
```

Comment:
Maybe it would be best to list specific columns and give them aliases, as event and envelope have some overlap when it comes to column names.

Reply:
Also, we need to filter out envelopes from bulkscan container. This is probably the best place to do it.